src/SeleniumLibrary/keywords/cookie.py

```diff
@@ -89,20 +120,19 @@ def add_cookie(self, name, value, path=None, domain=None, secure=None,
 
 
 class CookieInformation(object):
-    def __init__(self, cookie):
-        self.domain = cookie['domain'] if 'domain' in cookie else None
-        self.expiry = cookie['expiry'] if 'expiry' in cookie else None
-        self.httpOnly = cookie['httpOnly'] if 'httpOnly' in cookie else None
-        self.name = cookie['name'] if 'name' in cookie else None
-        self.path = cookie['path'] if 'path' in cookie else None
-        self.secure = cookie['secure'] if 'secure' in cookie else None
-        self.value = cookie['value'] if 'value' in cookie else None
+    def __init__(self, domain, expiry, httpOnly, name, path, secure, value):
+        self.domain = domain
+        self.expiry = expiry
+        self.httpOnly = httpOnly
+        self.name = name
+        self.path = path
+        self.secure = secure
+        self.value = value
 
     @property
     def full_info(self):
-        cookie_info = ', '.join("{}={}".format(key, value) for
-                                (key, value) in self.__dict__.items())
-        return cookie_info
+        return ', '.join("{}={}".format(key, value) for
+                         (key, value) in self.__dict__.items())
```

Comment:
1. As I already commented above, this property isn't too useful as `__str__` already shows all information.

2. Should have values in a predefined order, preferably in same order as passed to `__init__`. This ensures important information is first and also makes testing easier.

3. I'd also consider having all values on separate lines (i.e. joining with `\n` instead of `, `) to make easier to read the result.